apps/src/StudioApp.js

```diff
@@ -251,6 +251,15 @@ StudioApp.prototype.init = function(config) {
     config = {};
   }
 
+  if( config.isLegacyShare ) {
+    $("body").addClass("legacy-share-view");
+    if( dom.isIOS() ) {
+      if (!window.navigator.standalone) {
+        addToHome.show(true);
```

Comment:
Yes, on the ipad last week. Note that was only from my localhost so it was not behind CloudFront